File: app/src/main/java/com/vladislav/workoutassistant/data/db/entity/Set.java
package com.vladislav.workoutassistant.data.db.entity;

import com.vladislav.workoutassistant.data.models.RepeatableObject;

import androidx.room.ColumnInfo;
import androidx.room.Entity;

@Entity(tableName = "sets")
public class Set extends RepeatableObject {

    @ColumnInfo(name = "workout_id")
    private int mProgramId;

    public Set(int reps, int programId) {
        mReps = reps;
        mProgramId = programId;
    }

    /* GETTERS */
    public int getProgramId() {
        return mProgramId;
    }
}

File: app/src/main/java/com/vladislav/workoutassistant/data/db/entity/Set.kt
package com.vladislav.workoutassistant.data.db.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.vladislav.workoutassistant.data.models.Identifiable

@Entity(tableName = "sets")
data class Set(@PrimaryKey(autoGenerate = true) override val id: Int = 0,
               @ColumnInfo(name = "workout_id") val workoutId: Int,
               var amount: Int) : Identifiable